lib/__testfixtures__/replace-constructors.input.js

```diff
@@ -0,0 +1,3 @@
+const n = new Number();
```

Comment:
Note that these are all JS globals. We should also test (probably best in a separate file?) some web globals.